src/routes/hooks/entities/schemas/web-hook.schema.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod';
+import { ChainUpdateEventSchema } from '@/routes/hooks/entities/schemas/chain-update.schema';
+import { DeletedMultisigTransactionEventSchema } from '@/routes/hooks/entities/schemas/deleted-multisig-transaction.schema';
+import { ExecutedTransactionEventSchema } from '@/routes/hooks/entities/schemas/executed-transaction.schema';
+import { IncomingEtherEventSchema } from '@/routes/hooks/entities/schemas/incoming-ether.schema';
+import { IncomingTokenEventSchema } from '@/routes/hooks/entities/schemas/incoming-token.schema';
+import { MessageCreatedEventSchema } from '@/routes/hooks/entities/schemas/message-created.schema';
+import { ModuleTransactionEventSchema } from '@/routes/hooks/entities/schemas/module-transaction.schema';
+import { NewConfirmationEventSchema } from '@/routes/hooks/entities/schemas/new-confirmation.schema';
+import { NewMessageConfirmationEventSchema } from '@/routes/hooks/entities/schemas/new-message-confirmation.schema';
+import { OutgoingEtherEventSchema } from '@/routes/hooks/entities/schemas/outgoing-ether.schema';
+import { OutgoingTokenEventSchema } from '@/routes/hooks/entities/schemas/outgoing-token.schema';
+import { PendingTransactionEventSchema } from '@/routes/hooks/entities/schemas/pending-transaction.schema';
+import { SafeAppsUpdateEventSchema } from '@/routes/hooks/entities/schemas/safe-apps-update.schema';
+import { SafeCreatedEventSchema } from '@/routes/hooks/entities/schemas/safe-created.schema';
+
+export const WebHookSchema = z.discriminatedUnion('type', [
```

Comment:
Should we also rename this to `EventSchema` to stay consistent?

Reply:
Renamed when rebasing according to https://github.com/safe-global/safe-client-gateway/pull/1770#discussion_r1682730520.